README.md

```diff
@@ -1 +1,53 @@
-# Hello-Microverse-Project
\ No newline at end of file
+# Hello Microverse Project
+
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+## Live Demo
```

Comment:
Try to remove the empty headers it makes your `README.md` file not good looking and unprofessional